test/bdd/features/get-errors.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Get errors test
   Background: Setup local blockchain, bootstraps and nodes
-    Given the blockchain is set up
+    Given the blockchains are set up
```

Comment:
Maybe we should think about making those separate instructions, like:

```
Given the blockchain hardhat1 is set up
Given the blockchain hardhat2 is set up
```

Not the priority, but something to think about